### ISSUE TITLE
calwebb_tso3 and tso_photometry are both saving a _phot.ecsv output file

### ISSUE THREAD

**Issue by hbushouse** [STScI-JWST/jwst#1163]
Recent testing of the calwebb_tso3 pipeline on simulated NIRCam TSO imaging data results in:
```
2017-08-31 16:02:03,011 - stpipe.Tso3Pipeline.outlier_detection - INFO - Step outlier_detection done
2017-08-31 16:02:08,461 - stpipe.Tso3Pipeline - INFO - Writing Level 2c images with updated DQ arrays...
2017-08-31 16:02:09,424 - stpipe.Tso3Pipeline - INFO - Step.save_model jw12345001001_01101_00001_nrca1_crfints.fits
2017-08-31 16:02:34,646 - stpipe.Tso3Pipeline.tso_photometry - INFO - Step tso_photometry running with args (<jwst.datamodels.cube.CubeModel object at 0x1a9ab89d0>,).
2017-08-31 16:02:35,622 - stpipe.Tso3Pipeline.tso_photometry - INFO - Wrote TSO photometry catalog: jw12345001001_01101_00001_nrca1_phot.ecsv
2017-08-31 16:02:35,623 - stpipe.Tso3Pipeline.tso_photometry - INFO - Step tso_photometry done
2017-08-31 16:02:35,662 - stpipe.Tso3Pipeline - INFO - Step Tso3Pipeline done
```
which gives me the file ` jw12345001001_01101_00001_nrca1_phot.ecsv` created by the `tso_photometry` step and the file `jw12345-a3003_t1_nircam_f210m-clear_phot.ecsv` created by the calwebb_tso3 pipeline from the photometry table model returned by tso_photometry.

**Comment by hbushouse** [comment]
Also note that the `crfints` (level-2c) product file name does not contain the association candidate ID field anywhere. It should be inserted ahead of the `crfints` suffix, according to the latest level-2c file naming convention:

jw<ppppp><ooo><vvv>_<gg><s><aa>_<eeeee>_<detector>_<AC_ID>_<suffix>.fits

**Comment by stsci-hack** [comment]
The issue regarding the `crfints` (level-2c) product file name stems from the use of the .save_model() method of the Pipeline class.  @stscieisenhamer would be most familiar with this method and could investigate this aspect to provide details on what needs to be changed in the calling code in caltso3 to produce the correct level-2c filename.

**Comment by stsci-hack** [comment]
The two `phot.ecsv` files generated by `caltso3 `represent both exposure level products and a combined level-3 product containing all exposure level results in one table.  This functionality supports the case where the TSO observation was split into separate exposures (if I got the nomenclature correct).  If only one product should be generated, I will need to work with @larrybradley to work out how to turn off the generation of the exposure level results when collecting all the necessary results for the final level-3 product.  This would need to be handled in `tso_photometry `since the exposure level I/O occurs within the `tso_photometry `step with no option currently available to control whether or not a file gets written out.

**Comment by stsci-hack** [comment]
A quick discussion with @larrybradley confirmed that the catalog generated by the `tso_photometry `step represents the results for each input exposure.  The photometry table written out by the `caltso3` pipeline, on the other hand, merges all those individual catalogs into a master-combined catalog.  Should @hbushouse confirm that the individual catalogs not be generated (as originally implied), then the `tso_photometry` step can be easily modified to simply not call the `.write()` method to dump the catalog to a file.

**Comment by hbushouse** [comment]
I hereby confirm that the individual catalogs (from individual exposures) should not be generated.

**Comment by stsci-hack** [comment]
This issue has been addressed fully by PR #1180.
